database/stakes/db.go

```diff
@@ -27,6 +27,11 @@ FROM stake
          JOIN object_attribute USING (object_id)
 WHERE attribute_name = 'name'
   		AND wallet_id = $1`
+
+	getWallets = `SELECT wallet_id, contract_id, balance, blockchain_name, updated_at
```

Comment:
maybe just make the name of the query clearer getWalletsInfoQuery or something like this

Reply:
Agree, changed to `getWalletsInfoQuery`